src/widgets/mentors/ui/mentors.tsx

```diff
@@ -1,19 +1,22 @@
+import classNames from 'classnames/bind';
 import { LINKS } from '@/app/const';
 import mentorImg from '@/shared/assets/mentors-wanted.webp';
 import { Image } from '@/shared/ui/image';
 import { LinkCustom } from '@/shared/ui/link-custom';
 import { Paragraph } from '@/shared/ui/paragraph';
 import { WidgetTitle } from '@/shared/ui/widget-title';
 
-import './mentors.scss';
+import styles from './mentors.module.scss';
+
+const cx = classNames.bind(styles);
 
 export const Mentors = () => {
   return (
-    <section className="mentors container" id="mentors-wanted">
-      <div className="mentors content column-2">
-        <div className="mentors-info">
+    <section className={cx('container', 'mentors-container')} id="mentors-wanted">
+      <article className={cx('content', 'column-2', 'mentors-content')}>
```

Comment:
Let's first write the classes of the component itself, and then the global classes
`cx('mentors-container', 'container')` and so on

Reply:
we shouldn't manually do this, this must be done by `prettier`

Reply:
@Quiddlee i don't think it's smart enough to do that

Reply:
@SpaNb4 tailwindcss has that ability 

https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

Reply:
Done